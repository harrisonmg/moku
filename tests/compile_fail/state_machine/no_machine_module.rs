use moku::*;

#[state_machine]
mod blinky {
    use moku::*;

    enum BlinkyState {}
    impl StateEnum for BlinkyState {}

    struct Top {}
    impl TopState<BlinkyState> for Top {}
}

fn main() {}
