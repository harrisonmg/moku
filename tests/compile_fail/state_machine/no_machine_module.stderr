error: custom attribute panicked
 --> tests/compile_fail/state_machine/no_machine_module.rs:3:1
  |
3 | #[state_machine]
  | ^^^^^^^^^^^^^^^^
  |
  = help: message: not yet implemented
